packages/block-library/src/cover/edit.js

```diff
@@ -159,13 +221,16 @@ class CoverEdit extends Component {
 		};
 		const setDimRatio = ( ratio ) => setAttributes( { dimRatio: ratio } );
 
+		const { temporaryMinHeight } = this.state;
+
 		const style = {
 			...(
 				backgroundType === IMAGE_BACKGROUND_TYPE ?
 					backgroundImageStyles( url ) :
 					{}
 			),
 			backgroundColor: overlayColor.color,
+			minHeight: ( temporaryMinHeight || minHeight ) + 'px',
```

Comment:
I don't think we need px here, React assumes it's pixel if it's not set

Reply:
Suggestion applied 👍